Source/Core/DiscIO/Filesystem.h

```diff
@@ -86,6 +86,9 @@ class FileInfo
   // The size of a file.
   // Not guaranteed to return a meaningful value for directories.
   virtual u32 GetSize() const = 0;
+  // The total size of a file.
```

Comment:
This shouldn't say "file" without also mentioning "directory".

Maybe you could replace the whole comment (including the "guaranteed" part) with something like this: `For a file, returns its size. For a directory, returns the total size of its contents.`